include/cuda/std/detail/libcxx/include/array

```diff
@@ -137,19 +139,19 @@ struct _LIBCUDACXX_TEMPLATE_VIS array
     typedef const value_type*                     const_pointer;
     typedef size_t                                size_type;
     typedef ptrdiff_t                             difference_type;
-    typedef std::reverse_iterator<iterator>       reverse_iterator;
-    typedef std::reverse_iterator<const_iterator> const_reverse_iterator;
+    typedef std::reverse_iterator<const_iterator>      const_reverse_iterator;
+    typedef std::reverse_iterator<iterator>            reverse_iterator;
 
     _Tp __elems_[_Size];
 
     // No explicit construct/copy/destroy for aggregate type
     _LIBCUDACXX_INLINE_VISIBILITY void fill(const value_type& __u) {
-      _CUDA_VSTD::fill_n(__elems_, _Size, __u);
+      fill_n(__elems_, _Size, __u);
```

Comment:
This (and swap_ranges below) are now ADL calls, we need to keep them qualified.

Reply:
I've replaced `_CUDA_VSTD::` on this and below.